apps/api/routes/extract.py

```
@@ -0,0 +1,257 @@
+        use_case = get_extract_use_case()
+        result = await use_case.execute(limit=limit)
+
+        # Return response
+        return ExtractionResponse(
+            processed=result["processed"],
+            succeeded=result["succeeded"],
+            failed=result["failed"],
+        )
+
+    except Exception as e:
+        logger.error(f"Extraction pipeline failed: {e}", exc_info=True)
```

Ruff (0.14.1)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)